docker-compose-examples/keycloak-mssql.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.2"
+
+services:
+    mssql:
+        image: mcr.microsoft.com/mssql/server
+        volumes:
+            - type: volume
+              source: keycloakdbvolume
+              target: /var/opt/mssql
+        ports:
+            - "1433:1433"
+        environment:
+            - ACCEPT_EULA=Y
+            - SA_PASSWORD=Password!23
+            - MSSQL_PID=Developer
+    mssqlscripts:
+        image: mcr.microsoft.com/mssql-tools
+        depends_on:
+            - mssql
+        command: /bin/bash -c 'until /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P "Password!23" -Q "create database Keycloak"; do sleep 5; done'
+    keycloak:
+        image: jboss/keycloak
+        depends_on:
+            - mssql
+            - mssqlscripts
+        ports:
+            - "8080:8080"
+        environment:
+            - KEYCLOAK_USER=admin
+            - KEYCLOAK_PASSWORD=admin
+            - DB_VENDOR=mssql
+            - DB_USER=sa
+            - DB_PASSWORD=Password!23
+            - DB_ADDR=mssql
+            - DB_DATABASE=Keycloak
+volumes:
```

Comment:
Do we need this volumes? If not, please remove them.

Reply:
Volumes removed. They were present from the POCs for which this was being used.